addons/website/controllers/form.py

```diff
@@ -279,6 +278,25 @@ def insert_record(self, request, model, values, custom, meta=None):
                     message_type='comment',
                 )
 
+        if hasattr(record, '_message_log'):
+            warning_icon = ""
+            authenticate_message = False
+            if request.session.uid:
+                model_record = request.env[model_name]
+                user_email = request.env.user.email
+                form_email = values[model_record._mail_get_primary_email_field()]
```

Comment:
Indeed you right, thanks @rdeodoo 🙂

@ravb-odoo we could do that to avoid having a traceback because `_mail_get_primary_email_field` could return None if `_primary_email` attribute on the model is not defined or does not be a field inside the model.

```suggestion
        email_field_name = request.env[model_name]._mail_get_primary_email_field()
        if email_field_name and hasattr(record, '_message_log'):
            warning_icon = ""
            authenticate_message = False
            if request.session.uid:
                user_email = request.env.user.email
                form_email = values[email_field_name]
```

Note: I did not test my suggestion